nuxt.config.js

```diff
@@ -1,4 +1,19 @@
 export default {
+	// We need to override the default page name from index.vue to home.vue
+	// because we use the auto-generated routes for navigation and want
+	// the first page to be called "home"
+	router: {
+		extendRoutes(routes, resolve) {
+			routes.push({
+			path: '/',
+			name: 'home',
+			components: {
+			// eslint-disable-next-line no-undef
+			default: resolve(__dirname, 'pages/home.vue')
+			}
+		});
+		}
+	},
```

Comment:
Interesting, didn't know we could do this. Did u find this in their docs?

Reply:
it went a little like this:
step 1) ask chatgpt, step 2) confirm: https://v2.nuxt.com/docs/configuration-glossary/configuration-router/#extendroutes